views/lib/components/sc-card.hbs

```diff
@@ -47,7 +47,7 @@
                 {{/each}}
             </div>
         </div>
-        <div class="sc-card-body ckcontent">
+        <div class="sc-card-body ckcontent" data-testid="body-of-element">
```

Comment:
see above for consistent naming

Reply:
>title_of_an_element< is used as data-testid in the same file. Don't know the implications of changing data-testids that are in use.

Reply:
I actually meant to replace "body-of-element" to "body_of_element". There actually also is a testid already in this file named "header-of-element" which should also be changed to "header_of_element" for consistency. But maybe talk to Ankush if that is already used in several tests and the effort to change that would be too high.